paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -229,50 +214,55 @@ class Paparazzi @JvmOverloads constructor(
     val durationMillis = (end - start).toInt()
     val frameCount = (durationMillis * fps) / 1000 + 1
     val startNanos = TimeUnit.MILLISECONDS.toNanos(start)
-    takeSnapshots(view, name, deviceConfig, theme, renderingMode, startNanos, fps, frameCount)
+    takeSnapshots(view, name, startNanos, fps, frameCount)
   }
 
-  private fun takeSnapshots(
-    view: View,
-    name: String?,
+  fun unsafeConfig(
```

Comment:
nit: updateUnsafeConfig?

Reply:
How about `unsafeUpdateConfig`? The config itself isn't what's unsafe.

Reply:
True.  I'm down.